sdk/communication/azure-communication-email/samples/send_email_to_multiple_recipients_sample.py

```diff
@@ -37,7 +37,7 @@ class EmailMultipleRecipientSample(object):
 
     def send_email_to_multiple_recipients(self):
         # creating the email client
-        email_client = EmailClient.from_connection_string(self.connection_string)
+        email_client = EmailClient.from_connection_string(self.connection_string or "")
```

Comment:
```suggestion
        email_client = EmailClient.from_connection_string(self.connection_string)
```
We should avoid this change in the samples - it's not really valid.
Instead - replace the `os.getenv` statements on lines 33-36 to:
`connection_string = os.environ["COMMUNICATION_CONNECTION_STRING_EMAIL"]`

This way, the properties will not be `None` if they are not set - it will just fail. And that should avoid any mypy errors when you try to use them.

Reply:
Good call; it looks like that was being done on a few of the samples but not all. I updated them all as you suggested.